src/test/java/org/folio/rest/impl/InvoicesApiTest.java

```diff
@@ -146,26 +154,68 @@
   public void testGetInvoicingInvoices() {
     logger.info("=== Test Get Invoices by without query - get 200 by successful retrieval of invoices ===");
 
-    final InvoiceCollection resp = verifySuccessGet(INVOICE_PATH, InvoiceCollection.class);
+    final InvoiceCollection resp = verifySuccessGet(INVOICE_PATH, InvoiceCollection.class, X_OKAPI_PROTECTED_READ_TENANT);
 
     assertThat(resp.getTotalRecords(), is(3));
     assertThat(getInvoiceSearches(), hasSize(1));
     assertThat(getInvoiceLineSearches(), empty());
+    assertThat(MockServer.serverRqRs.get(ACQUISITIONS_UNITS, HttpMethod.GET), hasSize(1));
+    assertThat(MockServer.serverRqRs.get(ACQUISITIONS_MEMBERSHIPS, HttpMethod.GET), hasSize(1));
     verifyInvoiceUpdateCalls(0);
+
+    List<String> queryParams = getQueryParams(INVOICES);
+    assertThat(queryParams, hasSize(1));
+    assertThat(queryParams.get(0), Matchers.equalTo(getNoAcqUnitCQL(INVOICES)));
   }
 
   @Test
   public void testGetInvoicingInvoicesWithQueryParam() {
     logger.info("=== Test Get Invoices with query - get 200 by successful retrieval of invoices by query ===");
 
-    String endpointQuery = String.format("%s?query=%s==%s", INVOICE_PATH,  VENDOR_INVOICE_NUMBER_FIELD, EXISTING_VENDOR_INV_NO);
+    String sortBy = " sortBy folioInvoiceNo";
+    String queryValue = VENDOR_INVOICE_NUMBER_FIELD + "==" + EXISTING_VENDOR_INV_NO;
+    String endpointQuery = String.format("%s?query=%s%s", INVOICE_PATH,  queryValue, sortBy);
 
-    final InvoiceCollection resp = verifySuccessGet(endpointQuery, InvoiceCollection.class);
+    final InvoiceCollection resp = verifySuccessGet(endpointQuery, InvoiceCollection.class, X_OKAPI_PROTECTED_READ_TENANT);
 
     assertThat(resp.getTotalRecords(), is(1));
     assertThat(getInvoiceSearches(), hasSize(1));
+    assertThat(MockServer.serverRqRs.get(ACQUISITIONS_UNITS, HttpMethod.GET), hasSize(1));
+    assertThat(MockServer.serverRqRs.get(ACQUISITIONS_MEMBERSHIPS, HttpMethod.GET), hasSize(1));
     assertThat(getInvoiceLineSearches(), empty());
     verifyInvoiceUpdateCalls(0);
+
+    List<String> queryParams = getQueryParams(INVOICES);
+    assertThat(queryParams, hasSize(1));
+    String queryToStorage = queryParams.get(0);
+    assertThat(queryToStorage, containsString("(" + queryValue + ")"));
+    assertThat(queryToStorage, Matchers.not(containsString(ACQUISITIONS_UNIT_IDS + "=")));
+    assertThat(queryToStorage, containsString(getNoAcqUnitCQL(INVOICE)));
+    assertThat(queryToStorage, endsWith(sortBy));
+  }
+
+  @Test
+  public void testGetOrdersForUserAssignedToAcqUnits() {
```

Comment:
Since this is for an invoice, it should be something `testGetInvoiceForUserAssignedToAcqUnits`

Reply:
Corrected